edn/src/types.rs

```diff
@@ -24,7 +26,7 @@ pub enum Value {
     Float(OrderedFloat<f64>),
     Text(String),
     Symbol(String),
-    Keyword(String),
+    Keyword(keyword::Keyword),
```

Comment:
The latter "namespaced" refers to Rust's namespacing, not EDN's, right?  I think you could make `impl TryFrom<Into<String>> ...` and `Keyword(":foo/bar")` work.